test/e2e/cli/pkg_update_test.go

```diff
@@ -38,7 +38,7 @@ var _ = Describe("kraft pkg", func() {
 		var manifestsPath string
 
 		BeforeEach(func() {
-			cmd.Args = append(cmd.Args, "update")
+			cmd.Args = append(cmd.Args, "update", "--no-check-updates", "--log-level", "info", "--log-type", "json")
```

Comment:
Adding this everywhere isn't necessary, we disable it globally here:
https://github.com/unikraft/kraftkit/blob/5bea74c86bcde70f7ebfb315c86e005504352ab3/.github/workflows/gotests.yaml#L90